pdns/recursordist/docs/upgrade.rst

```diff
@@ -7,8 +7,42 @@ When upgrading several versions, please read **all** notes applying to the upgra
 4.2.x to 4.3.0 or master
 ------------------------
 
+Lua Netmask class methods changed
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 - Netmask class methods ``isIpv4`` and ``isIpv6`` have been deprecated in Lua, use :func:`Netmask.isIPv4` and :func:`Netmask.isIPv6` instead. In C++ API these methods have been removed.
 
+``socket-dir`` changed
+^^^^^^^^^^^^^^^^^^^^^^
+The default :ref:`setting-socket-dir` has changed to include ``pdns-recursor`` in the path.
+For non-chrooted setups, it is now whatever is passed to ``--with-socketdir`` during configure (``/var/run`` by default) plus ``pdns_recursor``.
```

Comment:
```suggestion
For non-chrooted setups, it is now whatever is passed to ``--with-socketdir`` during configure (``/var/run`` by default) plus ``pdns-recursor``.
```
Either I missed some complexity, or the directory always has a hyphen, not an underscore, right? (I think the _file_ has an underscore.)

Reply:
How I hate these gratuitous differences...